railties/test/railties/engine_test.rb

```diff
@@ -1315,6 +1315,8 @@ def index
       assert_equal "/foo/bukkits/bukkit", last_response.body
     end
 
+    # FIXME: Test requests are not made against the relative_url_root of "foo" - this only checks for the correct
```

Comment:
Why this FIXME note? Who is going to fix it?

Reply:
It is mostly because I thought the original test was actually verifying route generation was correct rather than just checking the url helper returned the correct path. I can remove the comment - thanks!